src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6716,3 +6716,141 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_unique_3d_with_duplicates_and_axis_2)
 
     test_case.run();
 }
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_blackmanwindow_periodic) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(ov::test::utils::getExecutableDirectory(),
+                                                                        SERIALIZED_ZOO,
+                                                                        "onnx/blackmanwindow_periodic.onnx"));
+
+    auto test_case = ov::test::TestCase(function, s_device);
+
+    test_case.add_input<int64_t>({10});
+    test_case.add_expected_output(Shape{10},
+                                  {-0.000000014901161f,
+                                   0.040212844f,
+                                   0.20077012f,
+                                   0.50978714f,
+                                   0.8492299f,
+                                   0.99999994f,
+                                   0.84922975f,
+                                   0.5097869f,
+                                   0.20077008f,
+                                   0.040212862f});
+
+    test_case.run_with_tolerance_as_fp();
+}
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_blackmanwindow_symmetric) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(ov::test::utils::getExecutableDirectory(),
+                                                                        SERIALIZED_ZOO,
+                                                                        "onnx/blackmanwindow_symmetric.onnx"));
+
+    auto test_case = ov::test::TestCase(function, s_device);
+
+    test_case.add_input<int64_t>({10});
+    test_case.add_expected_output(Shape{10},
+                                  {-0.00000001f,
+                                   0.05086961f,
+                                   0.25800052f,
+                                   0.63f,
+                                   0.9511299f,
+                                   0.9511298f,
+                                   0.62999994f,
+                                   0.25800028f,
+                                   0.05086958f,
+                                   -0.00000001f});
+
+    test_case.run_with_tolerance_as_fp();
+}
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_hammingwindow_periodic) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(ov::test::utils::getExecutableDirectory(),
+                                                                        SERIALIZED_ZOO,
+                                                                        "onnx/hammingwindow_periodic.onnx"));
+
+    auto test_case = ov::test::TestCase(function, s_device);
+
+    test_case.add_input<int64_t>({10});
+    test_case.add_expected_output<float>(Shape{10},
+                                         {0.0869565f,
```

Comment:
How did you get verification values? I generated slightly another set: {0.0869565f , 0.17414439f, 0.40240526f, 0.68455124f, 0.9128121f , 1.f, 0.9128121f, 0.6845511f, 0.4024052f, 0.17414442f}
I generate them by using python, for example:
```python
import numpy as np
np.set_printoptions(suppress=True,formatter=None,precision=16)
size = np.int32(10)
a0 = 25 / 46
a1 = 1 - a0
y = a0 - a1 * np.cos(2 * np.pi * np.arange(0, size, 1, dtype=np.float32) / size)
y
```
and it output:
array([0.0869565 , 0.17414439, 0.40240526, 0.68455124, 0.9128121 ,
       1.        , 0.9128121 , 0.6845511 , 0.4024052 , 0.17414442],
      dtype=float32)

Reply:
Understand. Yes, you are right, just update data using np.pi. Looks like it is the last blocker. I hope.